### ISSUE TITLE
[DOCS] Link to the Painless Lab in the "Painless Debugging" page

### ISSUE THREAD

**Issue by Ronserruya** [elastic/elasticsearch#69570]
The painless debugging page (https://www.elastic.co/guide/en/elasticsearch/painless/current/painless-debugging.html)
States that "Painless doesn’t have a REPL and while it’d be nice for it to have one day", while the Painless Lab is not a full REPL it's an excellent tool that provides a huge upgrade from the methods on that page.

I only found about the Painless Lab when I accidentally used a Kibana version that's newer than what we use in our org, and I've been using it almost daily since, it's a shame that it's hidden in the docs like this.

TL;DR Link to https://www.elastic.co/guide/en/kibana/master/painlesslab.html on https://www.elastic.co/guide/en/elasticsearch/painless/current/painless-debugging.html

Thanks again for all of the work on the lab ❤️

**Comment by cjcenizal** [comment]
Thanks so much for creating this issue @Ronserruya! Your kind words mean a lot to us. Great suggestion to improve visibility of Painless Lab -- we will get on this.

**Comment by lockewritesdocs** [comment]
Thanks for opening this issue @Ronserruya! I'm currently reworking the docs for [how to use scripts](https://www.elastic.co/guide/en/elasticsearch/reference/master/modules-scripting-using.html) so that they focus on teaching users how to write scripts in Painless. I'll make sure to explore and incorporate the Painless Lab 👍

**Comment by jdconrad** [comment]
Closing this as it is unlikely to be prioritized.
